python/cuml/cuml/tests/test_umap.py

```diff
@@ -869,3 +870,33 @@ def test_umap_trustworthiness_on_batch_nnd(
     cuml_trust = trustworthiness(digits.data, cuml_embedding, n_neighbors=10)
 
     assert cuml_trust > 0.9
+
+
+def test_callback():
```

Comment:
Moved this test to `test_umap.py` since it's more a test that UMAP calls the callback properly, not a test of the callback itself.